website/static/css/quick-project-search-plugin.css

```diff
@@ -0,0 +1,52 @@
+
+.node-styling {
+    background-color: #fcfcfc;
+    border: 1px solid #cccccc;
+}
+
+.node-styling:hover {
+    background-color: #E0EBF3;
+    cursor: pointer;
+}
+
+.quick-project button {
+    -webkit-appearance: none;
+    outline: none;
+    border: 0;
+    background: transparent;
+}
+
+.load-nodes {
+    font-size: 1.5em;
+}
+
+.quick-project .selected {
+    color: #000000
+}
+
+.quick-project .not-selected {
+    color: #347BB7;
+}
+
+.sort-group {
+  display: inline-block;
+
+}
+
+@media screen and (max-width: 768px) {
+    .node-col-headers {
+        display: none;
+    }
+}
+
+@media screen and (min-width: 768px) {
+    .node-sort-dropdown {
+        display: none;
+    }
+}
+
+label {
```

Comment:
As @caneruguz said [earlier](https://github.com/caneruguz/osf.io/pull/17#discussion_r50405417), a bit more specificity is preferred for selectors, to avoid overselecting

(P.S. This is a place where a [preprocessor](https://openscience.atlassian.net/browse/OSF-5524) can be helpful, because it allows one to [nest selectors](http://sass-lang.com/guide#topic-3) [though this can be [abused](www.sitepoint.com/beware-selector-nesting-sass/)].)